backend/README.md

```diff
@@ -33,3 +33,16 @@ All errors from backend should have in body:
   }
 }
 ```
+* Foce Update:
+	- error have to include a message to show in the alert
+	- error have to include an URL for the app redirect
+
+error response will be in the following format:
+```
+{
+  "errors": {
```

Comment:
@earayo but what I mean is that errors have two different formtats, one for errors where key is the reason of the error and another for force update with `url` and `message` field.